apps/client/src/common/utils/linkUtils.js

```diff
@@ -22,6 +24,7 @@ export function openLink(url) {
  * Handles opening external links
  * @param event
  * @param location
+ * @param alias (this is optional)
```

Comment:
thank you for keeping up the JSDocs

the convention for optional params is to put it inside square brackets, so 
`* @param [alias] `
This will also let the tooling understand that this is optional